src/Microsoft.AspNet.Routing/RouteValueDictionary.cs

```diff
@@ -31,28 +31,32 @@ public RouteValueDictionary()
         }
 
         /// <summary>
-        /// Creates a RouteValueDictionary initialized with the provided input values.
+        /// Creates a <see cref="RouteValueDictionary"/> initialized with the provided input value.
         /// </summary>
-        /// <param name="values">Input values to copy into the dictionary.</param>
-        public RouteValueDictionary([NotNull] IDictionary<string, object> values)
-        {
-            _dictionary = new Dictionary<string, object>(values, StringComparer.OrdinalIgnoreCase);
-        }
-
-        /// <summary>
-        /// Creates a RouteValueDictionary initialized with the provided input values.
-        /// </summary>
-        /// <param name="values">Input values to copy into the dictionary.</param>
+        /// <param name="obj">An object to initialize the dictionary. The value can be of type
+        /// <see cref="IDictionary{TKey, TValue}"/> or <see cref="IReadOnlyDictionary{TKey, TValue}"/> or
+        /// any other object.
+        /// </param>
         /// <remarks>
-        /// The input parameter is interpreted as a set of key-value-pairs where the property names
-        /// are keys, and property values are the values, and copied into the dictionary. Only public
-        /// instance non-index properties are considered.
+        /// If the value is a dictionary, then its entries are copied else the object is interpreted as a set
```

Comment:
Grammar fix:

`its entries are copied else the object is interpreted`
to
`its entries are copied. Otherwise the object is interpreted`